src/Microsoft.Dnx.Tooling/ConsoleCommands/ClearCacheConsoleCommand.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+
+namespace Microsoft.Dnx.Tooling
+{
+    internal static class ClearCacheConsoleCommand
+    {
+        public static void Register(CommandLineApplication cmdApp, ReportsFactory reportsFactory)
+        {
+            cmdApp.Command("clearcache", c =>
```

Comment:
Actually :-1/2: because it looks like we randomly do things in random places :smile: 

```
 -s|--source <FEED>
 -f|--fallbacksource <FEED>
 -p|--proxy <ADDRESS>
 --no-cache
 --packages
 --ignore-failed-sources
 --quiet
 --parallel
 -?|-h|--help
```

`--fallbacksource` is one word, but `--ignore-failed-sources` is multi-word.

Do whatever you want :smile: